### PR TITLE
Add MDM Intune integration mentions in documentation

### PR DESCRIPTION
**Pull request by TomasTurina**
## Description

This PR adds mentions to the new feature MDM Intune in `ms-graph` documentation.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/cloud-security/ms-graph/index.rst [hunk 1]
```diff
@@ -22,10 +22,11 @@ Currently, the module allows you to monitor the following with Wazuh:
 - Microsoft Defender for Office 365
 - Microsoft Purview eDiscovery
 - Microsoft Purview Data Loss Prevention (DLP)
+- Microsoft MDM Intune
 
 While these are centric to the security resource, the Microsoft Graph REST API contains a large number of additional resources that can be monitored. See the `Overview of Microsoft Graph <https://learn.microsoft.com/en-us/graph/overview?view=graph-rest-1.0>`_ documentation to learn more.
 
-.. note:: Currently, only the `security` resource can be considered mature as it's the only one tested and with pre-made rules. However, the logs of other resources can still be ingested at your organization's discretion.
+.. note:: Currently, only the `security` and `deviceManagement` resources can be considered mature as they're the only ones tested and with pre-made rules. However, the logs of other resources can still be ingested at your organization's discretion.
```
Comment (jr0me) [inline source/cloud-security/ms-graph/index.rst:29]: "tested with premade rules" or "tested, and that have premade rules"?

**Review by aritosteles — DISMISSED** [review]
LGTM

File: source/cloud-security/ms-graph/index.rst (hunk 1 above)
Comment (TomasTurina) [reply source/cloud-security/ms-graph/index.rst]: Done https://github.com/wazuh/wazuh-documentation/pull/7659/commits/f09284d0aaf9cb2aa1ae216c2b6463a39e9aaa03

**Review by aritosteles — APPROVED** [review]
LGTM

**Comment by vikman90** [comment]
Closed in favor of https://github.com/wazuh/wazuh-documentation/pull/7661.

**Pull request closed without merging**